app/ide-desktop/lib/dashboard/src/layouts/UserBar.tsx

```diff
@@ -68,26 +69,41 @@ export default function UserBar(props: UserBarProps) {
     <FocusArea active={!invisible} direction="horizontal">
       {innerProps => (
         <div
-          className="pointer-events-auto flex h-row shrink-0 cursor-default items-center gap-user-bar rounded-full bg-frame px-icons-x pr-profile-picture backdrop-blur-default"
+          className="pointer-events-auto flex h-row shrink-0 cursor-default items-center gap-1 rounded-full bg-frame px-icons-x pr-profile-picture backdrop-blur-default"
```

Comment:
Hmm, the idea of tailwind is using numeric values. I'm not sure using `semantic` values is a great idea at all.

Reply:
> the idea of tailwind is using numeric values

not quite. the idea of tailwind is using *predefined* values. semantic values are better here imo because we can change *all* spacings with a particular *semantic* meaning, in just one line. or in other words, the spacing can be determined by a theme

Reply:
>  spacing can be determined by a theme
I'm not sure this is what we want to achieve and worth spending our time on. 
IMO, 'w-1' is also semantic, because it doesn't specify what `1` is. We can define this using theme.
On the other hand prop line `gap-user-bar` doesn't give much clue on what is the user bar and where is it.

If we want to have `semantic` classnames, then maybe we should use CSS modules or `vanilla-extract`?

Defining a new selector for each component style is annoying and gives no benefits.

Reply:
Solution: we'll add tw variants lib https://www.tailwind-variants.org/docs/introduction